conf/application.conf

```diff
@@ -1,49 +1,124 @@
 # This is the main configuration file for the application.
```

Comment:
could you also link https://github.com/typesafehub/config in the top, so that people new to this configuration mechanism can look up the syntax and features?